preferences.py

```diff
@@ -180,6 +181,37 @@ def find_installed_rendermans(self, context):
         type=RendermanEnvVarSettings,
         name="Environment Variable Settings")
 
+    auto_check_update = bpy.props.BoolProperty(
+        name = "Auto-check for Update",
+        description = "If enabled, auto-check for updates using an interval",
+        default = False)
+    
+    updater_intrval_months = bpy.props.IntProperty(
+        name='Months',
+        description = "Number of months between checking for updates",
+        default=0,
+        min=0)
+
+    updater_intrval_days = bpy.props.IntProperty(
+        name='Days',
+        description = "Number of days between checking for updates",
+        default=7,
+        min=0)
+
+    updater_intrval_hours = bpy.props.IntProperty(
```

Comment:
I'd be fine hard coding this to once a day.  Or really why do they care about the interval time?  BTW is intrval a typo?  

I'd be fine if it just checked every time you loaded a file.